packages/swing-store/src/transcriptStore.js

```diff
@@ -257,18 +257,27 @@ export function makeTranscriptStore(
     WHERE vatID = ? AND position >= ? AND position < ?
   `);
 
-  function doSpanRollover(vatID, isNewIncarnation) {
+  function closeSpan(vatID, bounds) {
```

Comment:
Let's add a comment that `closeSpan` does not note any changes to the `.current` export and that the caller is in charge of that. An alternative would be to always note a deletion of that export when closing, but that's unnecessary churn for the common case.

Reply:
Done.